Fix the route line layer position issue after rerouting. (#4062)
mapbox/mapbox-navigation-ios: CHANGELOG.md

* Fixed an issue where the maneuver arrow on the route line overlapped labels for points of interest. ([#4030](https://github.com/mapbox/mapbox-navigation-ios/pull/4030))
* Fixed an issue where `NavigationMapView` injected with `NavigationOptions` isn't visible in active navigation. ([#4049](https://github.com/mapbox/mapbox-navigation-ios/pull/4049)) 
* Fixed the crash caused by showing a route that doesn't contain coordinates. ([#4046](https://github.com/mapbox/mapbox-navigation-ios/pull/4046))

> * Fixed an issue where the route line layer is above poi label. ([#4062](https://github.com/mapbox/mapbox-navigation-ios/pull/4062))

Reviewer: Nit: Most developers wouldn’t be familiar with the acronym “POI”:

```suggestion
* Fixed an issue where the route line layer appears above point of interest labels. ([#4062](https://github.com/mapbox/mapbox-navigation-ios/pull/4062))
```

Author: Thanks for the suggestion, the changeLog entry is updated.

* Fixed an issue where the maneuver arrow on the route line overlapped labels for points of interest. ([#4030](https://github.com/mapbox/mapbox-navigation-ios/pull/4030))
* Fixed an issue where `NavigationMapView` injected with `NavigationOptions` isn't visible in active navigation. ([#4049](https://github.com/mapbox/mapbox-navigation-ios/pull/4049)) 
* Fixed the crash caused by showing a route that doesn't contain coordinates. ([#4046](https://github.com/mapbox/mapbox-navigation-ios/pull/4046))

> * Fixed an issue where the route line layer appears above point of interest labels. ([#4062](https://github.com/mapbox/mapbox-navigation-ios/pull/4062))

Reviewer: We’ve already branched for v2.7.0, so let’s target v2.8.0 and move this changelog entry up to the “main” section unless we get feedback that it’s urgent enough to pull into v2.7.0.
